UPDATING.md

```diff
@@ -0,0 +1,22 @@
+# Upgrade / Migration Guide
+
+## Version 1.1.8 to 1.2.0
+
+### Notable Changes
+This release is all about channel options. Here a full [changelog](https://github.com/ably/ably-ruby/blob/main/CHANGELOG.md)
```

Comment:
Minor thing
Here a full [changelog](https://github.com/ably/ably-ruby/blob/main/CHANGELOG.md) -> Here is the full [changelog](https://github.com/ably/ably-ruby/blob/main/CHANGELOG.md)